app/javascript/controllers/show_more_button_controller.js

```diff
@@ -0,0 +1,46 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="show-more-button"
+export default class extends Controller {
+  connect() {
+    this.truncatedClass = 'content-truncated'
+    this.element.textContent = 'Show more'
+    this.element.classList.add('btn', 'btn-link', 'p-0', 'mb-3', this.truncatedClass, 'show-more-button')
+    this.element.ariaDisabled="true"
+    this.element.ariaExpaned="false"
+    this.element.ariaLabel = "This button is disabled because assistive technologies already announced the content."
+    this.element.prepend(this.buttonIcon())
+  }
+
+  truncated() {
```

Comment:
```suggestion
  isTruncated() {
```

I think this would be more idiomatic javascript.